server/src/weaverbird/backends/sql_translator/steps/utils/query_transformation.py

```diff
@@ -9,6 +9,7 @@
     Condition,
     ConditionComboAnd,
     ConditionComboOr,
+    DateBoundCondition,
```

Comment:
Pourquoi DateBound plutôt que RelativeDate ?

Reply:
This type correspond to the whole condition, so column + operator + value.
The value maybe either fixed or relative (so not always relative)